package.json

```diff
@@ -15,6 +15,7 @@
     "start": "yarn workspace @prosopo/provider-core run start",
     "fix-typescript-references": "utils-typescript-references",
     "test": "yarn workspace @prosopo/provider-core run test",
+    "test:mock": "./scripts/test.sh",
```

Comment:
`yarn test` is run from inside the provider container. 

`yarn test:mock` would need to be run from outside the container (as it messes with other containers and starts the provider container).

Maybe we should be using a different type of command to run this, such as `make mock` outside the container to populate the DBs, then start the docker running `yarn test:mock` as the piped command (`-c`). Thoughts?